master/buildbot/steps/artifact.py

```diff
@@ -469,11 +469,10 @@ def start(self):
         #find artifact dependency
         br = yield self._getBuildRequest()
 
-        if br["submitted_at"] > ARTIFACT_LOCATION_CHANGE_DATE:
-            artifactPath  = "%s/%s_%s" % (safeTranslate(self.artifactBuilderName),
-                                          br['brid'], FormatDatetime(br["submitted_at"]))
+        if self.customArtifactPath:
+            artifactPath = self.customArtifactPath
         else:
-            artifactPath  = "%s_%s_%s" % (safeTranslate(self.artifactBuilderName),
+            artifactPath  = "%s/%s_%s" % (safeTranslate(self.artifactBuilderName),
```

Comment:
Is it a real path to file? Maybe we should use `os.path.join` to ensure that we support both Windows and Linux path?

Reply:
It's a rsync URL